### DISCUSSION TITLE
Location of models on Win 10?

### DISCUSSION THREAD

**Discussion by nigelp** [Acly/krita-ai-diffusion#692] [Q&A] [answered]
Hi folks, where are the Diffusion AI models stored on Win 10? I can't find them anywhere on the disk in obvious folders. Cheers.

**Comment by lyonnel4** [comment, marked as answer]
In the config window in "styles", at the right of the "model checkpoint" line, you click on the folder icon and it will open a window at your models folder location.
Generally, it's in the  "comfyUI/models/checkpoints/" folder (that is where you installed the server)

**Reply by nigelp** [reply]
Thanks very much.
